modules/op-providing-secure-connection.adoc

```diff
@@ -0,0 +1,39 @@
+[id='op-providing-secure-connection_{context}']
+= Providing secure connection with OpenShift Routes
+
+You can create a route with the re-encrypt TLS termination:
+
+[source,terminal]
+----
+oc create route reencrypt --service=<svc-name> --cert=tls.crt --key=tls.key --ca-cert=ca.crt --hostname=<hostname>
+----
+
+Alternatively, you can create a re-encrypt TLS termination YAML file to create a secured route.
+
+.Example Re-encrypt TLS Termination YAML of the Secured Route
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  name: route-passthrough-secured  <1>
+spec:
+  host: <hostname>
+  to:
+    kind: Service
+    name: frontend <1>
+  tls:
+    termination: reencrypt <2>
+    key: [as in edge termination]
+    certificate: [as in edge termination]
+    caCertificate: [as in edge termination]
+    destinationCACertificate: |- <3>
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+----
+<1> The name of the object, which is limited to only 63 characters.
+<2> The termination field is set to re-encrypt. This is the only required TLS field.
```

Comment:
`reencrypt` within backticks. Also note the spelling in the code snippet - same should be applied here.